public/plugins/autosize-4.0.2/dist/autosize.js

```diff
@@ -0,0 +1,288 @@
+/*!
```

Comment:
`autosize.min.js` is the only file we need, other files should not be added.

Reply:
Removed.